apps/web/src/components/OccupationSelector.tsx

```diff
@@ -0,0 +1,26 @@
+import { useOccupations } from 'src/services/useOccupations';
+import { OccupationItem } from './OccupationItem';
+import { isOdd } from 'src/common/util';
+import { occupationItemBoxGray, occupationItemBoxWhite } from 'src/styles/styles';
+
+export const OccupationSelector = ({ selectedOccupations, setSelectedOccupations }: any) => {
+  const { occupations } = useOccupations();
+
+  return (
+    <div>
```

Comment:
This is supposed to make use of the Table component. But I'm ok this time. Kindly add a TODO to convert this to Table